### DISCUSSION TITLE
misc/physics/2

### DISCUSSION THREAD

**Discussion by giscus** [NoughtQ/notebook#25] [Announcements]
# misc/physics/2

NoughtQ的笔记本

https://note.noughtq.top/misc/physics/2

**Comment by 11111tao** [comment]
1/4 1/2波片的光程差分别是1/4\lambda 1/2\lambda

**Reply by 11111tao** [reply]
上面的那个也应该是相位差的公式

**Reply by NoughtQ** [reply]
sorry，本来早就想改的，做历年卷的时候就发现搞反了，之后我会更新的，感谢指出！

**Comment by 11111tao** [comment]
感谢lz的笔记！
